src/sql/azdata.proposed.d.ts

```diff
@@ -1434,18 +1434,26 @@ declare module 'azdata' {
 		}
 	}
 
-	export interface QueryPlanGraph {
+	export interface ExecutionPlanGraph {
 		/**
 		 * Root of the query plan tree
 		 */
-		root: QueryPlanGraphNode;
+		root: ExecutionPlanNode;
 		/**
 		 * Underlying query for the query plan graph.
 		 */
 		query: string;
+		/**
+		 * Raw string representation of graph
+		 */
+		graphFile: ExecutionPlanGraphFile;
+		/**
+		 * Query recommendations for optimizing performance
+		 */
+		recommendations: QueryPlanRecommendation[];
```

Comment:
let's be consistent, here it is query plan, other places we are renaming things to execution plan.